OMEdit/OMEditLIB/Modeling/Model.cpp

```diff
@@ -968,6 +968,17 @@ namespace ModelInstance
     return value.join(" ");
   }
 
+  QString Prefixes::typePrefixes() const
+  {
+    QStringList value;
+
+    if (!mConnector.isEmpty()) value.append(mConnector);
+    if (!mVariability.isEmpty()) value.append(mVariability);
+    if (!mDirection.isEmpty()) value.append(mDirection);
+
+    return value.join(" ");
```

Comment:
Call `value.removeAll(QString(""));`  before `value.join(" ")` otherwise we will get unnecessary spaces.